### PR TITLE
Allow .bgz/.bgz.tbi files to be read in TabixParser  

### PR DESCRIPTION
**Pull request by ima23**
## Description

Some resources use  _.bgz_ as ending for a BGZF-compressed vcf files (example gnomAD). Currently TabixParser allows only _.gz_ files to be read.

## Use case

The current changes will allow users to use _.vcf.bgz_ files for VEP custom annotations.

## Benefits

TabixParser will be able to process .bgz files and allow extended VEP custom annotations.

## Possible Drawbacks

_If applicable, describe any possible undesirable consequence of the changes._

## Testing

_Have you added/modified unit tests to test the changes?_

Yes.

_If so, do the tests pass/fail?_

Tests pass.

_Have you run the entire test suite and no regression was detected?_

Yes, tests passed.

### CODE REVIEW

File: modules/Bio/EnsEMBL/IO/TabixParser.pm [hunk 1]
```diff
@@ -49,7 +49,7 @@ sub open {
 
   my $self = $class->SUPER::new(@other_args);
 
-  confess "ERROR: Input file is not bgzipped, cannot use tabix\n" unless $filename =~ /\.gz$/;
+  confess "ERROR: Input file is not bgzipped, cannot use tabix\n" unless $filename =~ /\.gz$/ || $filename =~ /\.bgz$/;
```
Comment (tgrego) [inline modules/Bio/EnsEMBL/IO/TabixParser.pm]: What do you thing of providing the filename in the error message, and updating the regex so it does not need the or? (it is not a complex one...)
`print "ERROR: Input file $filename is not bgzipped, cannot use tabix\n" unless $filename =~ /\.b?gz$/;`

File: modules/t/vcf_tabix.t [hunk 2]
```diff
@@ -115,6 +115,16 @@ ok($parser->get_metadata_description('INFO', 'DP') eq 'Total Depth', 'getMetaDes
 
 ok ($parser->close(), "Closing file");
 
+note "\n> Testing reading bgz files:\n";
+my $test_file2 = $FindBin::Bin . '/input/data.vcf.bgz';
+my $parser2 = Bio::EnsEMBL::IO::Parser::VCF4Tabix->open($test_file2);
```
Comment (tgrego) [inline modules/t/vcf_tabix.t]: Could we test triggering that error by providing an unsupported file?

**Comment by ima23** [comment]
Thanks for the feedback @tgrego! I've added an update for the test and if clause.

File: modules/t/vcf_tabix.t (hunk 2 above)
Comment (ima23) [reply modules/t/vcf_tabix.t]: Done.

**Comment by ima23** [comment]
Thanks @tgrego! Would it be possible to have it in release/94 too? 
It's a small bug-fix to something that confuses users (.gz or .bgz for BGZF-compressed files) and community currently accepts both.

**Comment by tgrego** [comment]
sure no problem, will merge it now

**Pull request merged**